src/osp/Active/SysVehicle.cpp

```diff
@@ -87,6 +85,8 @@ ActiveEnt SysVehicle::activate(ActiveScene &rScene, universe::Universe &rUni,
     rScene.reg_emplace<ACompActivatedSat>(vehicleEnt, tgtSat);
 
     ACompVehicle& vehicleComp = rScene.reg_emplace<ACompVehicle>(vehicleEnt);
+    rScene.reg_emplace<ACompVehicleInConstruction>(
+                    vehicleEnt, loadMeVehicle.m_blueprint);
```

Comment:
ACompVehicleInConstruction here is added to newly created vehicles, to notify other systems to add their own components to the vehicle and/or its parts.

This is used by:
* `SysMachineContainer::update_containers`
* `SysMachineRCSController::update_construct`
* `SysMachineRocket::update_construct`
* `SysMachineUserControl::update_construct`

The component is removed at `SysVehicle::update_vehicle_modification`, at line 450.